util/request/helper.go

```diff
@@ -20,8 +19,7 @@ type Helper struct {
 // NewClient creates http client with default transport
 func NewClient(log *util.Logger) *http.Client {
 	return &http.Client{
-		Timeout:   Timeout,
-		Transport: NewTripper(log, transport.Default()),
+		Timeout: Timeout,
```

Comment:
This breaks logging

Reply:
Was removed by accident. I thought I added it in the beginning to debug my stuff. But it was there from the get go.